daisy_workflows/build-publish/enterprise_linux/centos_7_uefi.wf.json

```diff
@@ -0,0 +1,61 @@
+{
+  "Name": "centos-7-uefi",
+  "Project": "gce-image-builder",
+  "Zone": "us-central1-b",
+  "GCSPath": "gs://gce-image-build-bucket/daisy/${USERNAME}",
+  "Vars": {
+    "build_date": {
+      "Value": "${TIMESTAMP}",
+      "Description": "Build datestamp used to version the image."
+    },
+    "google_cloud_repo": {
+      "Value": "staging",
+      "Description": "The Google Cloud Repo branch to use."
+    },
+    "workflow_root": {
+      "Value": "/workflows",
+      "Description": "Root of github workflows, defaults to /workflows in the container."
+    },
+    "gcs-url": {
```

Comment:
use underscores for consistency: gcs_url (will need to be changed in all files and in concourse task config)